File: Class-work/OOP/account-app/src/com/monocept/model/test/AccountTest.java
package com.monocept.model.test;

import com.monocept.model.Account;

public class AccountTest {
	public static void main(String[] args)
	{
		Account a1 = new Account(100, "Faisal", 1000d);
		a1.deposit(5000d);
		a1.withdraw(4000);
		printDetails(a1);
		
		Account a2 = new Account(101, "Rohit");
		printDetails(a2);
	}
	
	public static void printDetails(Account a)
	{
		System.out.println("Account no. is = " + a.getAcc());
		System.out.println("Account holder name is = " + a.getName());
		System.out.println("Balance is = " + a.getBalance());
		System.out.println();
	}

}

File: NodeJs/module_basics/account_test.js
const Account = require("./account.js");
const a1 = new Account(101,"Faisal",500000);

a1.deposit(5000);

getDetails = function() {
    console.log("Id is : " + a1.getId());
    console.log("Name is : " + a1.getName());
    console.log("Balance is : " + a1.getBalance());
}

getDetails();